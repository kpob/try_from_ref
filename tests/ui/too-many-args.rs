#![no_main]

#[derive(derive_try_from::TryFromRef)]
#[source(SourceStruct)]
#[err(syn::Error)]
struct MyStruct {
    #[expr(1)]
    number: u32,
    #[default]
    #[expr("Hello".to_string())]
    text: String,
}