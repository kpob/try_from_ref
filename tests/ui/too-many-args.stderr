error: Cannot use default and expr attributes together

         = help: Remove one of them

  --> tests/ui/too-many-args.rs:9:5
   |
9  | /     #[default]
10 | |     #[expr("Hello".to_string())]
11 | |     text: String,
   | |________________^
